spec/registry.xml

```diff
@@ -0,0 +1,250 @@
+
+    </formatSignature>
+
+    <!-- TODO: IGNORED for now are the "Experimental" options. To discuss?
+      signDisplay@negative, useGrouping, roundingMode, roundingPriority, roundingIncrement, trailingZeroDisplay
+    -->
+
+  </function>
+
+  <function name="plural">
+    <!-- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/PluralRules
+      https://unicode-org.github.io/icu-docs/apidoc/dev/icu4j/com/ibm/icu/text/MessageFormat.html
+    -->
+    <description>Language-sensitive plural selection</description>
+    <matchSignature>
+      <option name="type" values="cardinal ordinal" default="cardinal"/>
+      <!--
+        Offset is neither ECMAScript Intl.PluralRules, nor ICU com.ibm.icu.text.PluralRules.
+        But it is implemented on top that base functionality.
+        Similar to exact values in number formatting, which does not come from NumberFormat(ter).
+        It is supported by both ICU and Fluent, and it is used for constructs like this:
+        "You invited A, B, and 2 more guests."
+      -->
+      <option name="offset" values="positiveInteger"/>
+      <!-- The minimum number of integer digits to use.
+        A value with a smaller number of integer digits than this number will be
+        left-padded with zeros (to the specified length) when formatted.
+      -->
+      <option name="minimumIntegerDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="1"/>
+      <!-- The minimum number of fraction digits to use.
+        The default for plain number and percent formatting is 0;
+        the default for currency formatting is the number of minor unit digits provided by
+        the ISO 4217 currency code list (2 if the list doesn't provide that information).
+      -->
+      <option name="minimumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
+      <!-- The maximum number of fraction digits to use.
+        The default for plain number formatting is the larger of minimumFractionDigits and 3;
+        the default for currency formatting is the larger of minimumFractionDigits and the number of minor
+        unit digits provided by the ISO 4217 currency code list (2 if the list doesn't provide that information);
+        the default for percent formatting is the larger of minimumFractionDigits and 0.
+      -->
+      <option name="maximumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
+      <!-- The minimum number of significant digits to use. -->
+      <option name="minimumSignificantDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="1"/>
+      <!-- The maximum number of significant digits to use. -->
+      <option name="maximumSignificantDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="21"/>
+
+      <match pattern="anyNumber"/>
+      <match values="zero one two few many"/>
```

Comment:
does * need to appear here? Some locales produce `other` as output from plural, which is effectively `*` in our syntax. That is, do we want to be able to say "this selector can request the default message"?

Reply:
I think it is an interesting minor issue for the registry spec.

Reply:
> I think our spec makes the presence of `*` mandatory. So I didn't include `other` for plural and the other selectors.

Why omit `other` if the spec only requires `*`?

Reply:
My thinking (why I excluded it): because they mean the same thing?
If I see `other` in the registry, I expect to also see `other` handled in the `when`

Reply:
This assumption is unfortunately false, likely on accident in CLDR, although I miss the context. For Polish, `other` is specifically used for fractions, while `many` is what you'd expect to be a good `other` in most other languages.